rpm/python-ramalama.spec

```diff
@@ -34,13 +36,19 @@ BuildRequires: golang
 BuildRequires: golang-github-cpuguy83-md2man
 BuildRequires: git-core
 BuildRequires: make
-BuildRequires: python%{python3_pkgversion}-devel
 BuildRequires: pyproject-rpm-macros
```

Comment:
I wonder if we take this out, will the problem disappear?

Reply:
trying

Reply:
@lsm5 as you see above the huggingface_hub[cli] as far as the RPM package is concerned is now fixed to only require hugginface_hub,  I have no idea why you are seeing the '[cli]' in your tests?

Reply:
let's see the latest run.

Reply:
I suspect the cli issue was still seen because of older builds existing in the copr. The latest build is still depending on omlmd probably because of its presence in  the toml file.

Reply:
rpm-macros is needed to recognize some python macros. So it'll have to be there. I can close this PR and open a new one to see cleaner test results if that helps. I think the only pain point now will be omlmd.